### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by trebmuh**
Udapte FR translation to 100%.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -568,38 +566,37 @@ msgid "U"
...
 
 #: ../src/GUI/ConfigDialog.cpp:53
 msgid "OSS MIDI Device"
-msgstr ""
+msgstr "Périphérique MIDI OSS"
 
 #: ../src/GUI/ConfigDialog.cpp:55
 msgid "Preferred Audio Driver"
-msgstr ""
+msgstr ""Pilote audio préféré"
```
Comment (nixxcode) [inline po/fr.po]: There is a problem here with too many `"`

**Comment by nixxcode** [comment]
Thanks, but there seems to be an error with the `fr.po` file:

```
../../po/fr.po:582: keyword "Pilote" unknown
../../po/fr.po:582:16: syntax error
../../po/fr.po:583: end-of-line within string
/usr/bin/msgfmt: found 3 fatal errors
```

On line 582 there seems to be too many `"` at the beginning.

**Comment by trebmuh** [comment]
fixed

File: po/fr.po (hunk 1 above)
Comment (trebmuh) [inline po/fr.po]: Thanks. Fixed.

**Pull request merged**